staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -223,6 +223,13 @@ func (o *ApplyOptions) Complete(f cmdutil.Factory, cmd *cobra.Command) error {
 	}
 	o.DryRunVerifier = resource.NewDryRunVerifier(o.DynamicClient, discoveryClient)
 	o.FieldManager = cmdutil.GetFieldManagerFlag(cmd)
+	if !cmd.Flag("field-manager").Changed {
+		if o.ServerSideApply {
+			o.FieldManager = "kubectl-server-side-apply"
```

Comment:
does changing the default mean the same kubectl serverside apply operation in subsequent releases would report a conflict with itself?

Reply:
Yes it does, that's a good point. We probably can't change the "server-side apply" default manager now, `kubectl` is still a fine name though.

Reply:
Ohh I see. Updated to `kubectl` and added a note about why